src/cli/subcommands/attach_cmd.rs

```diff
@@ -149,8 +151,6 @@ fn require(
                 .expect("as_object must succeed")
                 .get("exports", context);
 
-            // Reset module to avoid side effects
-            set_module(context);
```

Comment:
`module` cannot be redefined in `0.17`, any drawback in removing this?  @elmattic

Reply:
Hmm. I remember this was necessary for `require` but maybe just testing it still works without would be enough.

Reply:
@elmattic I've tested `showPeers()` and it worked. Do you mind checking out the branch and testing more? (I've no idea how to better test the feature)

Reply:
To test console module support can create a `calc.js` file with content:
```
module.exports = {
    add: function (a, b) {
      return a + b;
    },
    multiply: function (a, b) {
      return a * b;
    },
};

```
Then, once in the console:
```
> const Math = require("calc.js")
> console.log(Math.add(39,3));
42
```

Will test is on your branch now.

Reply:
Looks good!

Reply:
@elmattic Thanks! Should I go ahead and merge it?

Reply:
Yes.

Reply:
Thanks!